### ISSUE TITLE
Image attachment's bounds takes a String instead of CGRect

### ISSUE THREAD

**Issue by aheze** [malcommac/SwiftRichString#129]
The example in the README shows how to make an image, like this:

```swift
// You can specify the bounds of the image, both for size and the location respecting the base line of the text.
let localTextAndImage = AttributedString(image: UIImage(named: "rocket")!, bounds: CGRect(x: 0, y: -20, width: 25, height: 25))
```
But, when I try, it seems that `bounds` accepts a `String?`.

<kbd>![Screen Shot 2021-01-17 at 12 15 03 PM](https://user-images.githubusercontent.com/49819455/104854843-da550400-58bd-11eb-92e2-29e708d39bd3.png)</kbd>

<kbd>![Screen Shot 2021-01-17 at 12 17 30 PM](https://user-images.githubusercontent.com/49819455/104854866-fa84c300-58bd-11eb-8a88-7f125c5f149f.png)</kbd>

Shouldn't it take a `CGRect`?

**Comment by CherryPick62** [comment]
Yes! you can use this function "class func cgRect(for string: String) -> CGRect"  and "class func string(for: CGRect) -> String"
A string whose contents are of the form “{{x,y},{w, h}}”, where x is the x coordinate, y is the y coordinate, w is the width, and h is the height. These components can represent integer or float values. An example of a valid string is @”{{3,2},{4,5}}”. The string is not localized, so items are always separated with a comma.
https://developer.apple.com/documentation/foundation/nscoder/1624508-cgrect

**Comment by CherryPick62** [comment]
> The example in the README shows how to make an image, like this:
> 
> ```swift
> // You can specify the bounds of the image, both for size and the location respecting the base line of the text.
> let localTextAndImage = AttributedString(image: UIImage(named: "rocket")!, bounds: CGRect(x: 0, y: -20, width: 25, height: 25))
> ```
> 
> But, when I try, it seems that `bounds` accepts a `String?`.
> 
> ![Screen Shot 2021-01-17 at 12 15 03 PM](https://user-images.githubusercontent.com/49819455/104854843-da550400-58bd-11eb-92e2-29e708d39bd3.png)
> 
> ![Screen Shot 2021-01-17 at 12 17 30 PM](https://user-images.githubusercontent.com/49819455/104854866-fa84c300-58bd-11eb-8a88-7f125c5f149f.png)
> 
> Shouldn't it take a `CGRect`?

Yes! you can use this function "class func cgRect(for string: String) -> CGRect"  and "class func string(for: CGRect) -> String"
A string whose contents are of the form “{{x,y},{w, h}}”, where x is the x coordinate, y is the y coordinate, w is the width, and h is the height. These components can represent integer or float values. An example of a valid string is @”{{3,2},{4,5}}”. The string is not localized, so items are always separated with a comma.
https://developer.apple.com/documentation/foundation/nscoder/1624508-cgrect
